src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+                                    <VisualState x:Name="PointerOver">
+                                        <VisualState.Setters>
+                                            <Setter Target="LayoutRoot.Background" Value="{ThemeResource ListMediumControlBackgroundBrush}" />
+                                            <Setter Target="AccessibilityBorder.BorderBrush" Value="{ThemeResource ControlPointerOverBorderBrush}" />
+                                            <Setter Target="AccessibilityBorder.Background" Value="{ThemeResource ControlSelectedBackgroundBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                    <VisualState x:Name="Pressed">
+                                        <VisualState.Setters>
+                                            <Setter Target="LayoutRoot.Background" Value="{ThemeResource BaseMediumLowPressedBackgroundBrush}" />
+                                            <Setter Target="AccessibilityBorder.BorderBrush" Value="{ThemeResource ControlPointerOverBorderBrush}" />
+                                            <Setter Target="AccessibilityBorder.Background" Value="{ThemeResource ControlSelectedBackgroundBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                </VisualStateGroup>
+                                <VisualStateGroup x:Name="CheckStates">
+                                    <VisualState x:Name="Checked">
+                                        <VisualState.Setters>
+                                            <Setter Target="VisualBorder.BorderBrush" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="VisualBorder.Background" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="ContentPresenter.Background" Value="{ThemeResource BackgroundColorBrush}" />
+                                            <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource ForegroundColorBrush}" />
+                                        </VisualState.Setters>
+                                    </VisualState>
+                                    <VisualState x:Name="Unchecked" />
+                                    <VisualState x:Name="Indeterminate" />
+                                </VisualStateGroup>
+                            </VisualStateManager.VisualStateGroups>
+                        </Grid>
+                    </ControlTemplate>
+                </Setter.Value>
+            </Setter>
+        </Style>
+    </Page.Resources>
+
+    <Grid>
+        <StackPanel Margin="10,10,10,10">
+            <TextBlock x:Name="GridHeading" Text="Grid_d" FontSize="28"/>
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="30"/>
```

Comment:
You should use MinHeight here so that when the user changes their font size from accessibility settings the rows automatically change their height

Reply:
Fixed